feature/policy_forwarding/GUE/Outer_IPv4/decapsulation/otg_tests/Inner_IP/README.md

```diff
@@ -0,0 +1,337 @@
+# PF-1.4: Interface based GUE Decapsulation to IPv4 tunnel
+
+## Summary
+
+This is to test the the functionality of policy-based forwarding (PF) to decapsulate Generic UDP Encapsulation (GUE) traffic. These tests verify the use case of IPv4, and IPv6 encapsulated traffic in IPv4 GUE tuennel. The tests are meant for `Tunnel Interface` or `Policy Based` implementation of IPv4 GUE tunnel. The tests validate that the DUT performs the following action.
```

Comment:
What do tunnel interface and policy-based here? Are we allowing variance between the implementations -- note that we map a single policy-based approach to multiple underlying vendors already, and will require that in OpenConfig going forward...

Reply:
+1, we should focus on policy based.

The only need for  interface tunnels I think is if we want a control plane protocol  (ie: BGP) to have it's session encapsulated over a "fake interface" (tunnel).     Otherwise, we should use policy forwarding rules to determine encap/decap.

Underneath OC, implementations can translate the policy into their implementation specific requirements.

Reply:
Thanks Rob/Darren for taking time and sharing your input. I modified this section. Please take a look.